src/Illuminate/Console/GeneratorCommand.php

```diff
@@ -258,12 +261,14 @@ protected function alreadyExists($rawName)
      *
      * @param  string  $name
      * @return string
+     *
+     * @throws BindingResolutionException
```

Comment:
Can you please remove all these `@throws` annotations you added everywhere? These methods don't throw this exception, they only cascade.

Reply:
Removed...

Reply:
@imdhemy there's still some in this PR (including this one).